src/oal_agent/security/policies.py

```
@@ -4,11 +4,15 @@
 class SecurityPolicy:
     """Defines security policies."""
 
+    ALLOWED_ACTIONS = ["read", "write", "execute", "analyze"]  # Example allowed actions
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)